_data/simple-icons.json

```diff
@@ -12594,6 +12594,11 @@
             "hex": "F56640",
             "source": "https://wp-rocket.me/"
         },
+        {
+            "title": "WPExplorer",
+            "hex": "2563EB",
+            "source": "https://github.com/simple-icons/simple-icons/issues/5647#issuecomment-885079395"
```

Comment:
Incorrect link.

Reply:
Using the correct link `https://github.com/simple-icons/simple-icons/issues/5647#issue-885079395` is creating a linting error as the linter only accepts the `issuecomment`, not `issue` pattern in the regex.

As the issue is the original source of the file (and it was opened by WPExplorers account) I think this link would be the correct one as the source.

What would you suggest?

Reply:
@LitoMore I found three more icons where the source is pointing to an issue in this repo. My understanding of source is where the icon comes from - in this case this would be the linked issue as it was supplied by the company itself.

I checked the WPExplorer website but could not find any source for the used icon there.

But if it's requested I can of course just change it to [](https://wpexplorer.com)